inbm/dispatcher-agent/dispatcher/schedule/schedules.py

```diff
@@ -32,4 +32,17 @@ class RepeatedSchedule(Schedule):
     cron_month: str = field(default='*')
     cron_day_week: str = field(default='*')
     manifests: List[str] = field(default_factory=list)
-    
\ No newline at end of file
+
+@dataclass
+class SingleScheduleManifest:
+    """ Represents a SingleScheduleManifest object from the Scheduled Manifest """
+    priority: int
+    schedule_id: int
+    manifest_id: int
+
+@dataclass
+class RepeatedScheduleManifest:
+    """ Represents a RepeatedScheduleManifest object from the Scheduled Manifest """
```

Comment:
since these two dataclasses are the same, except in name.  We can just use ScheduledManifest for both to remove the redundancy.

Reply:
> since these two dataclasses are the same, except in name. We can just use ScheduledManifest for both to remove the redundancy.

the name might actually be important, as the meaning of the ints in these classes depend on which table they're coming from. when reading code, hovering over a variable of one of these types tells you what you can join it with.

Reply:
@gblewis1 - Normally you would create a base data class with the common fields and then create child classes with the specific name and any added fields.  We may try that, but there just wouldn't be any additional fields, and then we wouldn't have the redundancy.

Reply:
@nmgaston Actually here it looks like we don't need these particular classes since the `SELECT`s that are currently producing them should each be replaced by a `SELECT` with a `JOIN`. `get_all_single_schedule_in_priority` and `get_all_repeated_schedule_in_priority` are the methods holding the `SELECTS`.